Documentation: Start Page and Quickstart changes (#1588)
openvinotoolkit/model_server: docs/ovms_quickstart.md

```

> **NOTE**: For ONNX models additional steps are required. For a detailed description refer to our [ONNX format example](../demos/using_onnx_model/python/README.md).

OpenVINO Model Server expects a particular folder structure for models - in this case `model` directory has the following content: 
```bash
model/
└── 1
    ├── face-detection-retail-0004.bin
    └── face-detection-retail-0004.xml
``` 
Sub-folder 1 indicates the version of the model. If you want to upgrade the model, other versions can be added in separate subfolders (2,3...). 
For more information about the directory structure and how to deploy multiple models at a time, check out the following sections:
- [Preparing models](models_repository.md)
- [Serving models](starting_server.md)
- [Serving multiple model versions](model_version_policy.md) 

### Step 4: Start the Model Server Container

Start the container:

```bash
docker run -d -u $(id -u):$(id -g) -v $(pwd)/model:/models/face-detection -p 9000:9000 openvino/model_server:latest \

> --model_path /models/face-detection --model_name face-detection --port 9000

Reviewer: ```suggestion
--model_path /models/face-detection --model_name face-detection --port 9000 --shape auto
```
without shape auto parameter " python face_detection.py " will fail

> --model_path /models/face-detection --model_name face-detection --port 9000 --plugin_config '{"CPU_THROUGHPUT_STREAMS": "1"}' --shape auto

Reviewer 2: ```suggestion
--model_path /models/face-detection --model_name face-detection --port 9000
```

```

> During this step, the model folder is mapped to Docker. This folder will be used as the model storage from which the server will access models.

Reviewer 2: ```suggestion
During this step, the `model` folder is mounted to the Docker container. This folder will be used as the model storage from which the server will access models.
```

### Step 5: Prepare the Example Client Components

> Model scripts are available for quick access to the Model Server. Run an example command to download all required components:

Reviewer 2: ```suggestion
Client scripts are available for quick access to the Model Server. Run an example command to download all required components:
```
